Formula/ogre2.3.rb

```diff
@@ -30,6 +30,7 @@ def install
       "-DCMAKE_CXX_STANDARD=11",
       "-DCMAKE_CXX_STANDARD_REQUIRED:BOOL=ON",
       "-DCMAKE_CXX_EXTENSIONS:BOOL=ON",
+      "-DCMAKE_MACOSX_RPATH:BOOL=FALSE",
```

Comment:
we use `-DCMAKE_INSTALL_RPATH=#{rpath}` in our other formulae. I wonder if that would work for ogre2.3?

https://github.com/osrf/homebrew-simulation/blob/master/Formula/gz-sensors7.rb#L31

Reply:
that worked! I updated the PR to use this flag instead